### PR TITLE
Replace proprietary -moz by standard selector

### PR DESCRIPTION
**Pull request by MatonAnthony**
Replace proprietary -moz by standard selector to be compatible with a maximum of browser and with the W3C Standard.

### CODE REVIEW

File: lib/pre-without-class.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ exports.preWithoutClass = {
   name: "pre_without_class",
   desc: "pre_without_class_desc",
   check: function checkPreWithoutClass(rootElement) {
-    let presWithoutClass = rootElement.querySelectorAll("pre:-moz-any(:not([class]), [class=''])");
+    //let presWithoutClass = rootElement.querySelectorAll("pre:-moz-any(:not([class]), [class=''])");
+    let presWithoutClass = rootElement.querySelectorAll(":not([class]), [class='']");
```
Comment (Elchi3) [inline lib/pre-without-class.js]: Doesn't this match all elements without class? It should only match `<pre>` elements without class.

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: lib/pre-without-class.js (hunk 1 above)
Comment (MatonAnthony) [reply lib/pre-without-class.js]: Looks like I forgot a commit, yes it should only work on `<pre>`

Comment (MatonAnthony) [reply lib/pre-without-class.js]: I fixed it and added a test to ensure it doesn't happen again.

**Review by Elchi3 — APPROVED** [review]
Awesome!

**Pull request merged**